src/test/groovy/func/GradleCompatibilitySpec.groovy

```diff
@@ -25,26 +25,26 @@ class GradleCompatibilitySpec extends GatlingFuncSpec {
     }
 
     @Unroll
-    void 'should succeed for version #gradleVersion that greater than 4.0'() {
+    void 'should succeed for version #gradleVersion that greater than 5.0'() {
```

Comment:
Could you please elaborate why we have to drop Gradle 4 compat?

Reply:
Because of the [`named` method on `TaskCollection`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/TaskCollection.html#named-java.lang.String-)

> @Since 4.9

But didn't work as expected in 4.10.2 either.